3rdparty/fastcv/include/fastcv_hal_imgproc.hpp

```diff
@@ -88,33 +93,6 @@ int fastcv_hal_sobel(
     double          delta,
     int             border_type);
 
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-/// @brief Canny edge detector
-/// @param src_data Source image data
-/// @param src_step Source image step
-/// @param dst_data Destination image data
-/// @param dst_step Destination image step
-/// @param width Source image width
-/// @param height Source image height
-/// @param cn Number of channels
-/// @param lowThreshold low thresholds value
-/// @param highThreshold high thresholds value
-/// @param ksize Kernel size for Sobel operator.
-/// @param L2gradient Flag, indicating use L2 or L1 norma.
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-int fastcv_hal_canny(
-    const uchar*    src_data,
-    size_t          src_step,
-    uchar*          dst_data,
-    size_t          dst_step,
-    int             width,
-    int             height,
-    int             cn,
-    double          lowThreshold,
-    double          highThreshold,
-    int             ksize,
-    bool            L2gradient);
-
```

Comment:
Why is it removed? M.b. rebase issue?

Reply:
> Why is it removed? M.b. rebase issue?

It was added here by mistake, will be added in 3rd PR [opencv/opencv#26621](https://github.com/opencv/opencv/pull/26621)